staging/src/k8s.io/component-base/config/options/leaderelectionconfig.go

```diff
@@ -14,15 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package leaderelectionconfig
+/*
+NOTE: this file has been copied from k8s.io/kubernetes/pkg/client/leaderelectionconfig.
+*/
+package options
 
 import (
 	"github.com/spf13/pflag"
-	componentbaseconfig "k8s.io/component-base/config"
+	"k8s.io/component-base/config"
 )
 
 // BindFlags binds the LeaderElectionConfiguration struct fields to a flagset
-func BindFlags(l *componentbaseconfig.LeaderElectionConfiguration, fs *pflag.FlagSet) {
+func BindFlags(l *config.LeaderElectionConfiguration, fs *pflag.FlagSet) {
```

Comment:
For a future PR, I wonder if we should rename this to `BindLeaderElectionFlags` since it was moved to a generic package not specific to leader election.